packages/shared/web/src/dashboard-view/hooks/use-lend-borrow-list.tsx

```diff
@@ -29,17 +31,22 @@ export const useLendBorrowList = (
   const currentSeason = useCurrentSeason();
   const { baseCurrency } = useAppStore();
   const account = useAccountDefinition(network);
+  const currentNetworkStore = useCurrentNetworkStore();
+  const variableLend = currentNetworkStore.getPrimeCashLendData();
+  const variableDebt = currentNetworkStore.getPrimeCashDebtData();
+  const fCashLend = currentNetworkStore.getFCashLendData();
+  const fCashDebt = currentNetworkStore.getFCashDebtData();
+
   const isBorrow =
     product === PRODUCTS.BORROW_FIXED || product === PRODUCTS.BORROW_VARIABLE;
   const yieldDataKeys = {
-    [PRODUCTS.LEND_FIXED]: 'fCash',
-    [PRODUCTS.LEND_VARIABLE]: 'PrimeCash',
-    [PRODUCTS.BORROW_FIXED]: 'fCash',
-    [PRODUCTS.BORROW_VARIABLE]: 'PrimeDebt',
+    [PRODUCTS.LEND_FIXED]: fCashLend,
+    [PRODUCTS.LEND_VARIABLE]: variableLend,
+    [PRODUCTS.BORROW_FIXED]: fCashDebt,
+    [PRODUCTS.BORROW_VARIABLE]: variableDebt,
   };
-  const yieldData = useNetworkTokens(network, yieldDataKeys[product], {
-    isBorrow,
-  });
+
+  const yieldData = yieldDataKeys[product];
 
   let listColumns: DataTableColumn[] = [
```

Comment:
All of the `listColumns` in every one of these hooks is re-declared in every hook, which means it is created and destroyed on every render. Since it is used as a prop it should not trigger extra re-renders but as a pattern I think this adds unnecessary computation, lets move all of the column definitions out of the hooks and declare them as constants.

Reply:
Logic happens to remove columns based on data in the hooks. So that will still have to happen somewhere

Reply:
I do not see it as a performance issue

Reply:
Well in my opinion it also makes the code harder to read